metaflow/current.py

```diff
@@ -69,6 +70,31 @@ def is_running_flow(self):
     def flow(self) -> FlowSpec:
         return self._flow
 
+    def task_log_location(self, log_prefix: str, stream: Optional[str] = None) -> str:
+        """
+        This returns the current Task and Run datastore appropriate location
+        to store logs.
+
+        Args:
+            log_prefix (str): The prefix
+            stream (Optional[str], optional): If None, then the
+                When log_prefix == lightning_logs and stream==None:
+
+        Returns:
+            str: _description_
+
+        Examples:
+            Notice that the attempt number is prefixed to the log location path,
+            for example "0." on the first attempt, and on a retry the prefix
+            would be "1."
+
+            >>> current.log_location('lightning_logs')
+                ".metaflow/LightningCLIFlow/4666/start/29503/0.lightning_logs"
+            >>> current.log_location('lightning', 'out')
+                ".metaflow/LightningCLIFlow/4666/_parameters/29503/0.lightning_out.log"
```

Comment:
Did you want to change the path to be the step name and not `_parameters`?

Reply:
fixed thank you!